tools/github/scripts/Sync-AzsdkLabels.ps1

```diff
@@ -0,0 +1,82 @@
+[CmdletBinding(DefaultParameterSetName = 'RepositoryFile', SupportsShouldProcess = $true)]
+param (
+    [Parameter(Position = 0, Mandatory = $true)]
+    [ValidatePattern('^\w+/[\w-]+$')]
+    [string] $SourceRepository,
+
+    [Parameter(ParameterSetName = 'RepositoryFile')]
+    [ValidateScript({Test-Path $_ -PathType 'Leaf'})]
+    [string] $RepositoryFilePath = "$PSScriptRoot/../repositories.txt",
+
+    [Parameter(ParameterSetName = 'Repositories')]
+    [ValidateNotNullOrEmpty()]
+    [string[]] $Repositories = @(
+        'Azure/azure-sdk-for-cpp'
+        'Azure/azure-sdk-for-go'
+        'Azure/azure-sdk-for-java'
+        'Azure/azure-sdk-for-js'
+        'Azure/azure-sdk-for-net'
+        'Azure/azure-sdk-for-python'
+        'Azure/azure-sdk-tools'
+    ),
+
+    [Parameter(ParameterSetName = 'Languages')]
+    [ValidateNotNullOrEmpty()]
+    [string[]] $Languages = @('cpp', 'go', 'java', 'js', 'net', 'python'),
+
+    [Parameter()]
+    [switch] $Force
+)
+
+if (!(Get-Command -Type Application gh -ErrorAction Ignore)) {
+    throw 'You must first install the GitHub CLI: https://github.com/cli/cli/tree/trunk#installation'
+}
+
+if ($PSCmdlet.ParameterSetName -eq 'Languages') {
+    $Repositories = foreach ($lang in $Languages) {
+        "Azure/azure-sdk-for-$lang"
+    }
+}
+
+if ($PSCmdlet.ParameterSetName -eq 'RepositoryFile') {
+    $Repositories = Get-Content $RepositoryFilePath
+}
+
+# Filter out the source repository.
+$Repositories = $Repositories.Where({$_ -ne $SourceRepository})
+
+foreach ($repo in $Repositories) {
+    if ($Force -or $PSCmdlet.ShouldProcess(
+        "Cloning labels from $SourceRepository to $repo",
+        "Clone labels from $SourceRepository to $repo?",
+        "Clone labels")) {
+        $result = gh -R "$repo" label clone "$SourceRepository" --force 2>&1
```

Comment:
The `--force` here is to allow overwriting labels. If, for example, the description or color varies, it will be made consistent. I considered tying this to the scripts `-Force`, but that's really more for `ShouldProcess` to skip confirmation depending on the user's `$ConfirmPreference`. I could, though.

Reply:
Offline, @benbp was fine with this.